packages/theme/src/cli/utilities/theme-command.ts

```diff
@@ -102,7 +102,15 @@ export default abstract class ThemeCommand extends Command {
       return
     }
 
-    const environmentsMap = await this.loadEnvironments(environments, flags, klass)
+    const {flags: flagsWithoutDefaults} = await this.parse(noDefaultsOptions(klass), this.argv)
+    if ('path' in flagsWithoutDefaults) {
+      renderWarning({
+        body: "Provide each environment's command path in your shopify.theme.toml file.",
```

Comment:
Thinking out loud: what if I'm in `~` and I run `shopify theme command -e 1 -e2 --path ~/path/to/shopify.theme.toml`? I think that's probably the most common use case of including `--path` so we could potentially change the language of the warning maybe? Some pseudo code here:

```ts
if ('path' in flagsWithoutDefaults) {
  if (flagsWithoutDefaults.path === cwd()) {
    renderError({body: "Something about needing to run it in the same directory as the toml"});
    return;
  }

  renderError({body: "Cannot use the `--path` flag with multiple environments. Provide each environment's command path in your shopify.theme.toml file."});
  return
}
```

Reply:
And maybe use `renderError` since we can't continue?

Reply:
That makes sense, thanks! I updated the message